tests/consensus_tests/test_shard_stream_transfer_turbo4.py

```
@@ -0,0 +1,162 @@
+N_PEERS = 3
+COLLECTION_NAME = "test_collection"
+VECTOR_SIZE = 64
+NUM_POINTS = 200
+
+
+def _random_vector(rng):
+    return [rng.uniform(-1.0, 1.0) for _ in range(VECTOR_SIZE)]
+
+
+def create_turbo4_collection(peer_url):
+    r = requests.put(
```

Ruff (0.15.21)

[error] Probable use of `requests` call without timeout

(S113)